### PR TITLE
Update various go module dependencies

### PR DESCRIPTION
**Pull request by howardjohn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: pkg/tracing/config.go [hunk 1]
```diff
@@ -1,182 +0,0 @@
-// Copyright Istio Authors
```
Comment (howardjohn) [inline pkg/tracing/config.go:1]: There were some API changes that broke here. Its dead code though, so I just removed instead of fixing it

Comment (douglas-reid) [reply pkg/tracing/config.go:1]: I hadn't realized that this was no longer used, but it makes sense with Mixer no longer being around.

**Review by douglas-reid — APPROVED** [review]
Approving for P&T owned files.

**Comment by howardjohn** [comment]
@istio/wg-security-maintainers @istio/wg-test-and-release-maintainers can you take a look?

**Comment by linsun** [comment]
LGTM - can you resolve the conflict?

**Review by linsun — APPROVED** [review]

**Review by JimmyCYJ — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

File: tools/bug-report/pkg/bugreport/bugreport.go [hunk 2]
```diff
@@ -331,7 +331,9 @@ func BuildClientsFromConfig(kubeConfig []byte) (kube.Client, error) {
 
 func logAndPrintf(format string, a ...interface{}) {
 	fmt.Printf(format, a...)
-	log.Infof(format, a...)
+	o := []interface{}{format}
+	o = append(o, a...)
+	log.Infof(format, o)
```
Comment (hzxuzhonghu) [inline tools/bug-report/pkg/bugreport/bugreport.go:336]: This is not safe, say if one writes

`logAndPrintf("hello i am %s,  version %s", "istio", "1.8.0")`

**Pull request merged**